packages/newspringchurchapp/src/prayer/AddPrayer/AddPrayerHeader/index.js

```diff
@@ -0,0 +1,34 @@
+import React, { memo } from 'react';
+import { Image, View } from 'react-native';
+import PropTypes from 'prop-types';
+
+import { Avatar, styled, H3 } from '@apollosproject/ui-kit';
+
+const HeaderContent = styled(() => ({
+  alignItems: 'center',
+}))(View);
+
+const Title = styled(({ theme }) => ({
+  marginTop: theme.sizing.baseUnit * 0.5,
```

Comment:
We shouldn't need to add margin/padding to typography components.

Reply:
Agreed, but how do you suggest we space out components to match the design otherwise?

This is with no margins:
![header](https://user-images.githubusercontent.com/2659478/56504402-674f0b80-64e6-11e9-91db-3b27cf4ff527.png)

Reply:
I think for the header, which this is, it's okay to just use the `H3` without the `marginTop`. In this case it's probably closer to the mock without it.